apps/spack/build_spack.sh

```diff
@@ -24,33 +44,35 @@ git clone https://github.com/spack/spack.git
 cd spack
 git checkout tags/v${APP_VERSION}
 
-mkdir ${SPACKDIR}/spack/var/spack/repos/builtin/packages/hpcx
-cp ${APPS_SPACK_DIR}/package.py  ${SPACKDIR}/spack/var/spack/repos/builtin/packages/hpcx
```

Comment:
Will this work? How does spack know about hpcx without defining it?

Reply:
this is just replacing the flat file copy by a file structure